src/run.py

```diff
@@ -9,12 +10,28 @@
 kwargs = get_parameter()
 
 # check if a toolname was set in env
-toolname = os.environ.get('TOOL_RUN', 'whitebox_info').lower()
+toolname = os.environ.get('TOOL_RUN', 'fill_depressions').lower()
 
 # switch the tool
 if toolname == 'whitebox_info':
     wblib.print_info(to_file=kwargs.get('toFile', True))
 
+ # Fill Depression tool
+elif toolname == 'fill_depressions':
+    # get the parameters
+    try:
+        inp = kwargs['inputDEM']
+        out = kwargs['outputDEM']
+        flats = kwargs['fix_flats']
+    except Exception as e:
+        print(str(e))
+        sys.exit(1)
+
+    # run the whitebox fill_depression algorithm
+    print('Filling Depressions...')
+    wblib.fill(inp,out,flats)
```

Comment:
replace with:

```
print('Filling depressions...', end='')
wblib.fill(inp, out, flats)
print('done.')
```

This gives a bit nicer output and in Python you would usually add a whitespace after the comma :)

You could also extend the info like:

```Python
print(f"Filling depressions in DEM '{inp}'...")
```

to be even more expressive. This syntax of `f" {}"` is called string substitution and will replace the variable inside the curly braces with its string representation. Really handy.